pages/decision-reviews/higher-level-review.md

```diff
@@ -51,6 +51,35 @@ If you request an informal conference with a senior reviewer, they’ll call the
 
 ## How do I request a Higher-Level Review?
 
+Depending on the type of benefit claim, you can use our online Higher-Level Review request tool, submit a paper form by mail or fax, or drop it off in person at a regional benefit office near you.
+
+### Online
+TO request a Higher-Level Review for compensation or pension claims, you can use our online request tool. This online option is only available for compensation and pension claims. 
+
+<!-- Widget code -->
+<div data-widget-type="higher-level-review-status" data-widget-timeout="20">
+  <div class="loading-indicator-container">
+    <div class="loading-indicator" role="progressbar" aria-valuetext="Checking your application status."></div>
+    <span class="loading-indicator-message loading-indicator-message--normal">
+      Checking your application status.
+    </span>
+    <span class="loading-indicator-message loading-indicator-message--slow vads-u-display--none" aria-hidden="true">
+      Sorry, this is taking longer than expected.
+    </span>
+  </div>
+  <span class="static-widget-content vads-u-display--none" aria-hidden="true">
+  </span>
+    <div class="usa-alert usa-alert-error sip-application-error vads-u-display--none" aria-hidden="true">
+      <div class="usa-alert-body">
+        <strong>We’re sorry. Something went wrong when we tried to load your saved application.</strong><br>Please try refreshing your browser in a few minutes.
+      </div>
+    </div>
```

Comment:
The indention of this div block should line up with the span above. Other than that, it looks good!